server/utils/fileUtils.js

```diff
@@ -5,6 +5,8 @@ export const fileExtensionsArray = ['gif', 'jpg', 'jpeg', 'png', 'bmp', 'wav', '
   'mpeg', 'mpe', 'mpv', 'js', 'jsx', 'html', 'htm', 'css', 'json', 'csv', 'obj', 'svg',
   'otf', 'ttf', 'txt', 'mov', 'vert', 'frag'];
 
+export const textFileExtensionsArray = ['json', 'txt', 'csv', 'vert', 'frag', 'js', 'css', 'html', 'htm', 'jsx'];
```

Comment:
This is the same as `TEXT_FILE_REGEX`. We need an array to iterate over the extensions list. Also, this list is different from the Zeplin design. Is this the final list of extensions?

Reply:
Yes! The list of extensions in the Zeplin design are just placeholders.